biocypher_metta/adapters/epd_adapter.py

```diff
@@ -0,0 +1,65 @@
+import csv
+import gzip
+import pickle
+from biocypher_metta.adapters import Adapter
+# Example EPD bed input file:
+# chr1 959245 959305 NOC2L_1 900 - 959245 959256
+# chr1 960583 960643 KLHL17_1 900 + 960632 960643
+# chr1 966432 966492 PLEKHN1_1 900 + 966481 966492
+# chr1 976670 976730 PERM1_1 900 - 976670 976681
+
+class EPDAdapter(Adapter):
+    INDEX = {'chr' : 0, 'coord_start' : 1, 'coord_end' : 2, 'gene_id' : 3}
+
+    def __init__(self, filepath, hgnc_to_ensembl_map, write_properties, add_provenance, type='propmoter', label='promoter', delimiter=' '):
+        self.filepath = filepath
+        self.hgnc_to_ensembl_map = pickle.load(open(hgnc_to_ensembl_map, 'rb'))
+        self.type = type
+        self.label = label
+        self.delimiter = delimiter
+
+        self.source = 'EPD'
+        self.version = '006'
+        self.source_url = 'https://epd.expasy.org/ftp/epdnew/H_sapiens/'
+
+        super(EPDAdapter, self).__init__(write_properties, add_provenance)
+
+    def get_nodes(self):
+        with gzip.open(self.filepath, 'rt') as f:
+            reader = csv.reader(f, delimiter=self.delimiter)
+            for line in reader:
+                chr = line[EPDAdapter.INDEX['chr']]
+                coord_start = int(line[EPDAdapter.INDEX['coord_start']]) + 1
+                coord_end = int(line[EPDAdapter.INDEX['coord_end']]) + 1
+                promoter_id = chr+":"+str(coord_start)+"-"+str(coord_end)
```

Comment:
You should import and use the [build_regulatory_region_id](https://github.com/Habush/biocypher-metta/blob/04e7b97f314485e9f56735849dcab3cdf673f61f/biocypher_metta/adapters/helpers.py#L35) function from the helpers script.